config/initializers/content_security_policy.rb

```diff
@@ -112,7 +112,7 @@
 # ###############
 
 # If you are using UJS then enable automatic nonce generation
-Rails.application.config.content_security_policy_nonce_generator = -> request { SecureRandom.base64(16) }
+Rails.application.config.content_security_policy_nonce_generator = ->(_request) { SecureRandom.base64(16) }
```

Comment:
can you remove the arg entirely here? or is it required

Reply:
I think keeping the arg helps as documentation so future dev doesn't have to go lookup the docs to see that the request is passed in. That being said, I'm happy to go with what the consensus of reviews is here.

Reply:
I'm not that fussed either, but if I had to pick I'd favor removing it because I don't think it's likely we're going to ever write our own nonce generator